python/jupytergis_core/pyproject.toml

```diff
@@ -20,7 +20,6 @@ classifiers = [
 ]
 dependencies = [
   "jupyter-ydoc>=2,<3",
-  "jupyter-collaboration>=3,<4",
```

Comment:
Should we use extras instead?

Technically, this break jupytergis-core, right?

Reply:
> Technically, this break jupytergis-core, right?

Does `jupytergis-core` use `jupyter-collaboration` specifically?

Reply:
Yes, it provides the labextension for opening JupyterGIS files in JupyterLab

Reply:
But it just needs an `ICollaborativeDrive`, right?

Reply:
Then someone needs to provide it. If we remove the jupyter-collaboration from jupytergis-core, nobody provides collaboration support when we just do:

```
pip install jupytergis-core
```

Reply:
That is the goal of this PR, so that someone can provide their own collaboration system (other than jupyter-collaboration).
Since the jupyter-collaboration was moved to `jupytergis` you still get it on a default installation.

Reply:
Ok. Yeah I guess it's not possible to do that using extras...